tests/unit/lib/intrinsic_resolver/test_intrinsic_resolver.py

```diff
@@ -731,10 +734,7 @@ def test_fn_and_arguments_invalid_formats(self, name, intrinsic):
             self.resolver.intrinsic_property_resolver({"Fn::And": intrinsic}, True)
 
     @parameterized.expand(
-        [
-            ("Fn:And must have all arguments that resolves to booleans".format(item), item)
-            for item in [{}, 42, None, "test"]
-        ]
+        [(f"Fn:And must have all arguments that resolves to booleans {item}", item) for item in [{}, 42, None, "test"]]
```

Comment:
is there a reason, we are doing this change just here? do we need a larger sweep?

Reply:
mypy complaints about this because we were calling `.format` on a string that does not have any placeholder. So I fixed them all. Because f-string are less prune to such error, I used it.

Reply:
yes, I meant is it just these files, but as you said, looks like mypy only complained about these files.